backend/src/main/resources/graphiql/renderExample.js

```diff
@@ -11,139 +11,150 @@
+  let headers = opts.headers;
+  // Convert headers to an object.
+  if (typeof headers === 'string') {
+    headers = JSON.parse(opts.headers);
+  }
 
-    return fetch(api, {
-        method: 'post',
-        headers: Object.assign(
-          {
-              Accept: 'application/json',
-              'Content-Type': 'application/json',
-          },
-          headers,
-        ),
-        body: JSON.stringify(graphQLParams),
-        credentials: 'omit',
-    })
+  return fetch(api, {
+    method: 'post',
+    headers: Object.assign(
+        {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        headers,
+    ),
+    body: JSON.stringify(graphQLParams),
+    credentials: 'omit',
+  })
       .then(function (response) {
-          return response.text();
+        return response.text();
       })
       .then(function (responseBody) {
-          try {
-              return JSON.parse(responseBody);
-          } catch (error) {
-              return responseBody;
-          }
+        try {
+          return JSON.parse(responseBody);
+        } catch (error) {
+          return responseBody;
+        }
       });
 }
 
+function SelectGraphqlInstance() {
+  const [endpoint, setEndpoint] = React.useState(null)
+  if (endpoint === null) {
+    return React.createElement(React.Fragment, {
+      children: [
+        React.createElement("h4", {children: "Choose Endpoint!"}),
+        React.createElement("button", {children: "stores", onClick: () => setEndpoint("stores")}),
+        React.createElement("button", {children: "verification", onClick: () => setEndpoint("verification")}),
+      ]
+    })
+  }
+  return React.createElement(GraphiQL, {
+    fetcher: graphQLFetcher(endpoint),
+    query: parameters.query,
+    variables: parameters.variables,
+    headers: parameters.headers,
+    operationName: parameters.operationName,
+    onEditQuery: onEditQuery,
+    onEditVariables: onEditVariables,
+    onEditHeaders: onEditHeaders,
+    defaultSecondaryEditorOpen: true,
+    onEditOperationName: onEditOperationName,
+    headerEditorEnabled: true,
+  })
+}
+
+
 // Render <GraphiQL /> into the body.
 // See the README in the top level of this module to learn more about
 // how you can customize GraphiQL by providing different values or
 // additional child elements.
-ReactDOM.render(
-  React.createElement(GraphiQL, {
-      fetcher: graphQLFetcher,
-      query: parameters.query,
-      variables: parameters.variables,
-      headers: parameters.headers,
-      operationName: parameters.operationName,
-      onEditQuery: onEditQuery,
-      onEditVariables: onEditVariables,
-      onEditHeaders: onEditHeaders,
-      defaultSecondaryEditorOpen: true,
-      onEditOperationName: onEditOperationName,
-      headerEditorEnabled: true,
-  }),
-  document.getElementById('graphiql'),
-);
+ReactDOM.render(React.createElement(SelectGraphqlInstance), document.getElementById('graphiql'));
```

Comment:
Why is there another call to `React.createElement` necessary? Does not `SelectGrapphqlInstance` already create an element?

Reply:
`SelectGraphqlInstance` itself is only a function which cannot be passed to `ReactDOM.render`.

Reply:
Ah, of course, thanks for the explanation 👍🏽